python/lsst/ts/standardscripts/maintel/base_close_loop.py

```diff
@@ -371,23 +381,23 @@ async def handle_cwfs_mode(self) -> None:
 
         # Run WEP
         await self.mtcs.rem.mtaos.cmd_runWEP.set_start(
-            visitId=visit_id, timeout=2 * CMD_TIMEOUT
+            visitId=visit_id, timeout=2 * CMD_TIMEOUT, config=self.wep_config
         )
 
     async def compute_ofc_offsets(self) -> None:
         """Compute offsets using ts_ofc."""
 
         # Create the config to run OFC
         config = {
-            "filter_name": self.filter,
+            "filter_name": "",
```

Comment:
Are you sure you want to remove the filter here? Was this updated in a different PR?